e2e/tests/transfer/base_test.go

```diff
@@ -7,54 +7,67 @@ import (
 	"testing"
 	"time"
 
-	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
 	"github.com/strangelove-ventures/interchaintest/v8/ibc"
 	test "github.com/strangelove-ventures/interchaintest/v8/testutil"
 	testifysuite "github.com/stretchr/testify/suite"
 
 	sdkmath "cosmossdk.io/math"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
-	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 
 	"github.com/cosmos/ibc-go/e2e/testsuite"
 	"github.com/cosmos/ibc-go/e2e/testsuite/query"
 	"github.com/cosmos/ibc-go/e2e/testvalues"
 	transfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
-	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
 func TestTransferTestSuite(t *testing.T) {
 	testifysuite.Run(t, new(TransferTestSuite))
 }
 
 type TransferTestSuite struct {
-	testsuite.E2ETestSuite
+	transferTester
 }
 
-func (s *TransferTestSuite) SetupTest() {
-	s.SetupPaths(ibc.DefaultClientOpts(), s.TransferChannelOptions())
+// transferTester defines some helper functions that can be used in various test suites
+// that test transfer functionality.
+type transferTester struct {
```

Comment:
this is a construct to share functionality of transfer tests across different test suites.